lib/clearance/password_strategies/bcrypt.rb

```diff
@@ -6,6 +6,9 @@ module PasswordStrategies
     # will perform) is automatically set to the minimum allowed value when
     # Rails is operating in the test environment and the default cost in all
     # other envionments. This provides a speed boost in tests.
+    #
+    # To set your own cost value, use an initializer:
```

Comment:
This assumes knowledge of BCrypt that most people won't have, so I'd think we should explain this value a bit more.

```suggestion
  # BCrypt has a `cost` argument which determines how computationally expensive
  # the hash is to calculate. The higher the cost, the harder it is for
  # attackers to crack passwords even if they posess a database dump of the
  # encrypted passwords. Clearance uses the `bcrypt-ruby` default cost except in 
  # the test environment, where it uses the minimum cost value for speed. If you
  # wish to increase the cost over the default, you can do so by setting a
  # higher cost in an initializer: 
```